Road-To-Ricochet-2.md

```diff
@@ -0,0 +1,52 @@
+
+
+# Ideal integration Scenario:
+## Rational:
+
+Because clients can't use AK/SK to run CI, if clients creates a Pull Request, they can't run CI tests.
+
+As sure, it would be ideal to treat clients and Outscale developers at the same level.
```

Comment:
I don't know if `as sure` is possible, I know `as sure as`. Otherwise just put `Of course`  or `For sure`